metadata/packagesruntime.json

```diff
@@ -2020,7 +2020,497 @@
         "command": "./run-kotor1.sh",
         "engine_name": "reone"
     },
-    "39510": {
+    "34270": {
+    "game_name": "SEGA Mega Drive & Genesis Classics",
+    "download": [
+        {
+            "name": "blastem",
+            "url": "https://github.com/luxtorpeda-dev/packages/releases/download/blastem/",
+            "file": "blastem-34270.tar.xz"
+        }
+    ],
+    "command": "./run-blastem.sh",
+    "information": {
+        "store_link": "https://store.steampowered.com/app/34270/",
+        "engine_name": "blastem",
+        "engine_link": "https://www.retrodev.com/blastem/",
+        "version": "0.6.3-PRE",
+        "comments": "",
+        "author": "kunio",
+        "author_link": "https://github.com/ToughGuyKunio",
+        "license": "GPLv3",
+        "license_link": "https://www.retrodev.com/repos/blastem/file/tip/COPYING",
+        "non_free": false,
+        "closed_source": false
```

Comment:
This should be moved to the new way of providing the information, using the common object (see vkquake or another engine for how that works). 

That would need to happen for each instance of this game engine being used. You don't need the comments line in that case, and you can remove the non_free and closed_source items since they are false.

Reply:
Like this?